### PR TITLE
Fix "Overweight" trait unintended interaction

### PR DESCRIPTION
**Pull request by Varoxus**
## About The Pull Request
The hunger hud update changed how the Overweight trait is handled, making it give any character using it the speed malus that the trait already gives, but also giving them an extra speed malus as if they were overfed at all times. Currently there's no way to get rid of the permanently overfed state without manually removing the "Overweight" trait directly from the mob that has it, this PR removes that permanent overfed state.

## Why It's Good For The Game
I don't think anyone taking the overweight trait was supposed to be both slowed from the trait itself and forced into the overfed state permanently, so this just solves that issue.

## Changelog
:cl:
fix: Overweight trait no longer permanently makes a character overfed
/:cl:

### CODE REVIEW

**Comment by Varoxus** [comment]
> Ehm, what's the "overweight" trait? All this PR seems to do is comment out code.

The ""TRAIT_FAT" bit, but now that I look at it... the trait that used to give you that isn't actually in there in the current version of the... So this ain't actually a fix, dang. Well! Least it already got fixed.

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -799,9 +799,11 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/splash)
 		state = HUNGER_STATE_FINE
 		return
 
+	/*
 	if(HAS_TRAIT(hungry, TRAIT_FAT))
 		state = HUNGER_STATE_FAT
 		return
+	*/
```
Comment (LemonInTheDark) [inline code/_onclick/hud/screen_objects.dm:806]: Shouldn't just comment out code, if we want code back we have git to retrieve it. No sense leaving a mess you know?

Comment (LemonInTheDark) [reply code/_onclick/hud/screen_objects.dm:806]: Just figured I'd say that straight for the future

Comment (Varoxus) [reply code/_onclick/hud/screen_objects.dm:806]: Ah it's fine to just delete it? Well, I shall keep that in mind for the future! Thanks much.

**Pull request closed without merging**